### PR TITLE
Release package version 1.8.0

### PR DESCRIPTION
**Pull request by lorenzwalthert**
Prepare for release:

* [x] `devtools::check_win_devel()`
* [x] [Polish NEWS](http://style.tidyverse.org/news.html#before-release)
* [x] add tidy thanks.
* [x] If new failures, update `email.yml` then `revdepcheck::revdep_email_maintainers()`
* [x] update `cran-comments.md`
* [x] run `urlchecker::url_check("path/to/pkg")`

Perform release:

* [x] Create RC branch (for bigger releases)
* [x] Bump version (in DESCRIPTION and NEWS)
* [x] add tidy thanks to `NEWS.md`
* [ ] `devtools::submit_cran()`
* [x] `pkgdown::build_site()`
* [ ] Approve email

Wait for CRAN...

* [ ] Tag release
* [ ] Merge RC back to master branch
* [ ] Bump dev version
* [ ] update CRAN speed change monitor PR.We have a new CRAN release coming up 🥳 See [`NEWS.md`](https://styler.r-lib.org/dev/news/index.html) for new features.

### CODE REVIEW

**Comment by lorenzwalthert** [comment]
@IndrajeetPatil can you have a look? In particular to the `NEWS.md` for spelling mistakes etc. Once approved, I can submit.

File: NEWS.md
Comment (IndrajeetPatil) [inline NEWS.md:61]: Is this supposed to be `styler.colored_print.vertical`?

**Review by IndrajeetPatil — APPROVED** [review]
Overall, it looks good!

The only thing that bothers me is that some of the news items start with uppercase letters, while others with lowercase. I was going to change it, but NEWS says that we should not edit it by hand. So not sure. 

But this is not something that should block the release. We can punt it to a later date.

**Comment by IndrajeetPatil** [comment]
It's on CRAN! 🎉

**Comment by lorenzwalthert** [comment]
Yes. 🥳 I will sort out the git tagging and then merge. I guess we could also write a blog post on tidyverse.com and summarize all new features since the [last post](https://www.tidyverse.org/blog/2019/11/styler-1-2-0/). Are you interested to contribute to that?

**Comment by IndrajeetPatil** [comment]
Sure, would love to contribute!

**Comment by MichaelChirico** [comment]
Great work!! Thank you both!!

**Pull request merged**
